Classes/Notifications/NotificationSectionController.swift

```diff
@@ -86,9 +86,13 @@ final class NotificationSectionController: ListSwiftSectionController<Notificati
         case .hash(let hash):
             viewController?.presentCommit(owner: model.owner, repo: model.repo, hash: hash)
         case .number(let number):
+            let model = IssueDetailsModel(owner: model.owner, repo: model.repo, number: number)
+            if let controller = currentDetailController, controller.model == model {
+                return
+            }
             let controller = IssuesViewController(
```

Comment:
Would be better to change the name to `issuesViewController`, because atm there are two controllers with the same name.

Reply:
Done